docs/_sources/entry_points/faq.md.txt

```diff
@@ -45,3 +45,7 @@ Not yet, but we are working on it. In the meantime, you try our [examples](../ge
 ## [setup] I need help for the setup and deployment, can you help me?
 
 Please reach us [here](../getting_started/installation/local_install_skaffold.md##need-help).
+
+## [usage] What if I gave access to a dataset with a too wide set of permissions and I change my mind?
+
+As the dataset is hosted on your side, you can always remove or de-reference it to make it unreachable.
```

Comment:
What do you mean by "de-referencing" ? Is there a way to do this in Substra ?

Reply:
No, not yet, this is why it is suggested to remove the dataset in this case; shall I rephrase?

Reply:
I think it would be more appropriate to say "delete the local files", as we can't really remove or de-reference the dataset on Substra

Reply:
Fair enough, thanks for your insight @Fabien-GELUS

Reply:
You might want to detail the procedure @natct10 ;)

Reply:
Like how to erase the files? I am not sure that it needs to be that specific as it is a FAQ question, don't you think?

Reply:
Still, I have opened an issue about your point @chrisalexandrepena : #80.
Feel free to add any comments!